src/App.jsx

```diff
@@ -1,35 +1,32 @@
-import Home from "./containers/Home/Home";
-import userProfile from "./assets/data/dummyUserData"
+// import {useState} from "react";
 import "./App.scss";
-import SearchContainer from "./containers/SearchContainer/SearchContainer";
 import Button from "./components/Button/Button";
+import SearchContainer from "./containers/SearchContainer/SearchContainer";
+
+// TEMPORARY //////////////////
+import PathwaysCard from "./components/PathwaysCard/PathwaysCard";
+
+const card = {header: "Financial Forest", topics: "Banking, Investments"};
```

Comment:
Put this into the data folder.

Reply:
Hey Charlie, are you only referring to line 9? I think line 7 should stay in App.jsx?